lib/chef/provider/aws_rds_instance.rb

```diff
@@ -0,0 +1,49 @@
+require 'chef/provisioning/aws_driver/aws_provider'
+
+class Chef::Provider::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSProvider
+  REQUIRED_OPTIONS = %w(db_instance_identifier allocated_storage engine
+                        db_instance_class master_username master_user_password)
+
+  OTHER_OPTIONS = %w(engine_version multi_az iops publicly_accessible db_name db_port)
+
+  def update_aws_object(instance)
+    Chef::Log.warn("aws_rds_instance does not support modifying a started instance")
+  end
+
+  def create_aws_object
+    converge_by "Create new RDS instance #{new_resource.db_instance_identifier} in #{region}" do
+      new_resource.driver.rds.client.create_db_instance(create_options)
+    end
+  end
+
+  def destroy_aws_object(instance)
+    converge_by "Deleting RDS instance #{new_resource.db_instance_identifier} in #{region}" do
+      instance.delete(skip_final_snapshot: true)
+    end
+  end
+
+  def create_options
+    opts = {}
+    opts.merge(new_resource.additional_options)
+    REQUIRED_OPTIONS.each do |opt|
+      opts[opt] = new_resource.send(opt.to_sym)
+    end
+    OTHER_OPTIONS.each do |opt|
+      opts[opt] = new_resource.send(opt.to_sym) if ! new_resource.send(opt.to_sym).nil?
+    end
+    opts[:tags] = hash_to_tag_array(new_resource.aws_tags) if new_resource.aws_tags
```

Comment:
I don't think coverge_tags will work.  It does the following:

https://github.com/chef/chef-provisioning-aws/blob/master/lib/chef/provisioning/aws_driver/aws_provider.rb#L256

But tags isn't even a method on AWS::RDS::DBInstance:

```
irb(main):001:0> require 'aws'
=> true
irb(main):002:0> r = AWS::RDS::DBInstance.new("foo", {})
=> <AWS::RDS::DBInstance db_instance_identifier:foo>
irb(main):003:0> r.tags
NoMethodError: undefined method `tags' for <AWS::RDS::DBInstance db_instance_identifier:foo>:AWS::RDS::DBInstance
        from (irb):3
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/cli/console.rb:38:in `run'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/cli.rb:286:in `console'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/vendor/thor/command.rb:27:in `run'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/vendor/thor/invocation.rb:121:in `invoke_command'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/vendor/thor.rb:363:in `dispatch'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/vendor/thor/base.rb:440:in `start'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/cli.rb:9:in `start'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/bin/bundle:20:in `block in <top (required)>'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/lib/bundler/friendly_errors.rb:5:in `with_friendly_errors'
        from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.7.12/bin/bundle:18:in `<top (required)>'
        from /opt/chefdk/embedded/bin/bundle:23:in `load'
        from /opt/chefdk/embedded/bin/bundle:23:in `<main>'
```